cardano-node/src/Cardano/Node/Types.hs

```diff
@@ -355,6 +376,17 @@ data NodeHardForkProtocolConfiguration =
        -- configured the same, or they will disagree.
        --
      , npcTestShelleyHardForkAtVersion :: Maybe Word
+
+       -- TODO @node-team: We should probably define other types such as
+       -- 'ShelleyHardForkProtocolConfiguration',
+       -- 'AllegraHardForkProtocolConfiguration', and
+       -- 'MaryHardForkProtocolConfiguration'.
```

Comment:
What I mean by this is that we should probably have fields of those types within `NodeHardForkProtocolConfiguration`.

Reply:
I don't think we need to at this point but it's up to you @intricate